vars/main.yml

```diff
@@ -109,3 +109,5 @@ os_security_suid_sgid_system_whitelist:
 
 # system accounts that do not get their login disabled and pasword changed
 os_always_ignore_users: ['root', 'sync', 'shutdown', 'halt']
+
+proc_hidepid_entry: 'proc                    /proc                   proc    defaults,nosuid,nodev,noexec,relatime,hidepid=2,gid=root    0 0'
```

Comment:
We should probably make the hidepid variable configurable. Then we also ne a need entry in the readme and in the tests.

Reply:
Do you mean like such a true/false variable?

Reply:
Yes!
Maybe also a hidepid variable to set it to `0`, `1` and `2`? What do you think?

Reply:
Ok, good idea. Do you think is it better to create a new file like `mount.yml` to add all kind of future fstab hardenization or leave for the moment all in `minimize_access.yml`?

Reply:
Leave it there for now. :)